ItaCH_Smash_Legends/Assets/Script/LegendController.cs

```diff
@@ -0,0 +1,187 @@
+        _animatorOverride = new AnimatorOverrideController(_anim.runtimeAnimatorController);
+        _anim.runtimeAnimatorController = _animatorOverride;
+
+    }
+    private void Start()
+    {
+        for (int i = 0; i < _action.Length; ++i)
+        {
+            _action[i] = _input.actions[_actionLiteral[i]];
+        }
+    }
+    private void Update()
+    {
+    }
+    private void OnMove(InputValue value)
+    {
+        Vector2 input = value.Get<Vector2>();
+
+        if (input != null)
+        {
+            MoveDirection = new Vector3(input.x, 0, input.y);
+            _anim.SetBool(AnimationHash.Run, true);
+        }
+    }
+
+    private void OnJump() { }
+    private void OnDefaultAttack() { }
+    private void OnSmashAttack() { }
+    private void OnSkillAttack() { }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.CompareTag("HangZone"))
+        {
+            // Animation.Play(Hang);
+        }
+    }
+
+    public void NextAnimation()
+    {
+        if (OnActionTrigger(ActionType.Move))
+        {
+            _anim.Play(AnimationHash.Run);
+        }
+        if (OnActionTrigger(ActionType.Jump))
+        {
+            _anim.Play(AnimationHash.Jump);
+        }
+        if (OnActionTrigger(ActionType.DefaultAttack))
+        {
+            _anim.Play(AnimationHash.FirstAttack);
+        }
+        if (OnActionTrigger(ActionType.HeavyAttack))
+        {
+            _anim.Play(AnimationHash.HeavyAttack);
+        }
+        if (OnActionTrigger(ActionType.SkillAttack))
+        {
+            _anim.Play(AnimationHash.SkillAttack);
+        }
+    }
+    public void PlayFirstAttack()
+    {
+        if (SetAttackable(ActionType.DefaultAttack))
+        {
+            _anim.Play(AnimationHash.FirstAttack);
+            AttackRotate();
+        }
+    }
+    public void PlaySecondAttack()
+    {
+        if (SetAttackable(ActionType.DefaultAttack))
+        {
+            _anim.Play(AnimationHash.SecondAttack);
+            AttackRotate();
+        }
+    }
+    public void NextPlayClip()
+    {
+        if (PossibleComboCount < ApplyClip.Length - 1)
+        {
+            _animatorOverride[_animationClipLiteral[PossibleComboCount]] = ApplyClip[PossibleComboCount];
+            ++PossibleComboCount;
+        }
+    }
+
+    private bool OnActionTrigger(ActionType type)
+    {
+        if (_action[(int)type].triggered)
+        {
+            return true;
+        }
+        else
+        {
+            return false;
+        }
+    }
+    public bool SetAttackable(ActionType type)
+    {
+        if (OnActionTrigger(type) && PossibleComboCount < ApplyClip.Length)
+        {
+            return true;
+        }
+        else
+        {
+            return false;
+        }
+    }
+    public bool SetAttackable()
+    {
+        if (_isAttack)
+        {
+            return true;
+        }
+        else
+        {
+            return false;
+        }
+    }
+    public void NextComboPossible()
```

Comment:
코드 컨벤션 맞지 않음.

Reply:
아래의 메소드까지 합해서

```suggestion
public void SetComboPossible(bool isPossible) => _isAttack = isPossible;
```
가 좋아보임.